tests/YahooWeatherClientTests.php

```diff
@@ -10,11 +10,314 @@
 
 namespace Th3Mouk\YahooWeatherAPI\Tests;
 
+require 'src/Query/Query.php';
```

Comment:
I just want to quick testing so I use require to load the all necessary PHP files.